docs/index.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Flow Client Library (FCL)
+---
+<br />
+<p align="center">
+  <h1 align="center"> FCL JS</h1>
+  <p align="center">
+    <i>FCL JS is the best way to connect your dapp to users, their wallets and Flow.</i>
+    <br />
+    <a href="https://docs.onflow.org/fcl-js/"><strong>Read the docs»</strong></a>
+    <br />
+    <br />
+    <a href="https://docs.onflow.org/fcl/tutorials/flow-app-quickstart/">Quickstart</a>
+    ·
+    <a href="https://github.com/onflow/fcl-js/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/onflow/fcl-js/blob/master/CONTRIBUTING.md">Contribute</a>
+
+  </p>
+</p>
+<br />
+<br />
+
+## What is FCL?
+
+The Flow Client Library (FCL) is a package used to interact with user wallets and the Flow blockchain. When using FCL for authentication, dapps are able to support all FCL-compatible wallets on Flow and their users without any custom integrations or changes needed to the dapp code.
+
+FCL was created to make developing JavaScript applications that connect to the Flow blockchain easy and secure. It defines a standardized set of communication patterns between wallets, applications, and users that is used to perform a wide variety of actions for your dapp. FCL also offers a full featured SDK and utilities to interact with the Flow blockchain.
+
+FCL JS is the javascript implementation of FCL and can be used in both browser and server environments. All functionality for connecting and communicating with wallet providers is restricted to the browser.
```

Comment:
Ah, seems like we want to use "FCL JS" - maybe a simple illustration or explanation would help: FCL is a concept. FCL JS implements this concept for JS. Other implementations are: FCL Swift, ...

Reply:
for sure, updated the last bit with some of this!